vispy/scene/cameras/turntable.py

```diff
@@ -23,14 +23,16 @@ class TurntableCamera(Base3DRotationCamera):
     fov : float
         Field of view. Zero (default) means orthographic projection.
     elevation : float
-        Elevation angle in degrees. Positive angles place the camera
-        above the cente point, negative angles place the camera below
-        the center point.
+        Elevation angle in degrees. The elevation angle represents a
+        rotation of the camera around the current scene x-axis. The
+        camera looks along the +y direction when the elevation is 0 degrees.
     azimuth : float
-        Azimuth angle in degrees. Zero degrees places the camera on the
-        positive x-axis, pointing in the negative x direction.
+        Azimuth angle in degrees. The azimuth angle represents a
+        rotation of the camera around the scene z-axis according to the
+        right-hand screw rule.
```

Comment:
Can/should you mention what the camera "sees" when azimuth is at 0 degrees as it did in the old docstring?